docs/api/response.md

```diff
@@ -17,7 +17,7 @@
 
 ### response.socket
 
-  Request socket.
```

Comment:
actually `res.socket` and `req.socket` are point to the same `net.Socket` instance.

Reply:
Made another PR to reflect this fact: https://github.com/koajs/koa/pull/1357.    :alien: